doc/component/form-select.md

```diff
@@ -1,2 +1,35 @@
 # form-select
-[TBD]
+The `<form-select />` component allows you to render a select block.
+
+## Attributes
+
+The following attributes (bindables) are available for this component.
+
+| attribute | type | default | description |
+|---|---|---|---|
+| name | string | `''` | The name of the select |
+| classes | string | `''` | Classes to add |
+| readonly | boolean | `false` | Specifies that the select should be read-only |
+| disabled | boolean | `false` | Specifies that the select should be disabled |
+| multiple | boolean | `false` | Make the select a multi-select block |
+| selectOptions | array | `[]` | The selected options |
+| options | array/object | `{}` | The available options |
+| optionLabel | string | `'name'` | Which attribute to show in the option |
+| autofocus | boolean | `false` | Specifies to enable the autofocus of the select |
+| required | boolean | `false` | Make the select required |
+
+## Example
+```html
+<aurelia-select
+    name="message"
+    classes="form-control"
+    readonly.bind="false"
+    disabled.bind="false"
+    multiple.bind="true"
+    selectOptions="[1,3]"
+    options.bind="[{id: 55}, {id: 120}, {id: 99}]"
```

Comment:
Only `selectOptions ` should be used to specified the options in the select. `options` is something else. It is used in combination with the customAttribute `prefixed` to add prefix to the classes of the element. 

I think the bindable `options` is actually not necessary here and can be removed. 

@RWOverdijk, agreed?